src/types/SignTransaction.ts

```diff
@@ -0,0 +1,23 @@
+// TODO: Dummy type for ballot transaction. Should be remove when implement logic from core
```

Comment:
Could you please move this file to:

`src/ui/pages/IncomingRequest/components/SignTransactionRequest.types.ts`